packages/builder/src/components/backend/DatasourceNavigator/TableIntegrationMenu/rest/auth/RestAuthenticationModal.svelte

```diff
@@ -28,7 +41,16 @@
   let hasErrors = false
   let hasChanged = false
 
-  onMount(() => {
+  let createVariableModal
+  let formFieldkey
+
+  onMount(async () => {
+    await environment.loadVariables()
```

Comment:
The other `loadVariables` calls have a try catch wrapper, if this or the licensing init were to fail it would likely cause issues on the page load.